forth_src/test.fs

```diff
@@ -1,6 +1,7 @@
 : fakekeys ( n -- ) $c6 c! ;
 
-here latest @
+$cbff top! \ this section compiles past $6fff.
```

Comment:
I think it's an ugly wart here, that the v edit buffer sits as $7000 and grows unbounded.

Long term, I would like to see v replaced with a much simpler text editor that uses a fixed amount of RAM for the edit buffer, I think ideally no more than $1000 bytes, maybe even $400 would be fine (the number of characters that will fit on a screen). Personally I do not think it makes sense to edit huge text files on C64 anyway.

There are many interesting ideas for interactive Forth text editing historically, so it's pick and choose.